CHANGELOG.md

```diff
@@ -107,6 +107,8 @@
 - Improved documentation of the `readme` option, including multiple files and additional formats ([#5158](https://github.com/python-poetry/poetry/pull/5158))
 - Improved contributing documentation ([#5708](https://github.com/python-poetry/poetry/pull/5708))
 - Remove all references to `--dev-only` option ([#5771](https://github.com/python-poetry/poetry/pull/5771))
+- Added environment variable checking for PyPi token ([#5911](https://github.com/python-poetry/poetry/pull/5911))
```

Comment:
You are editing the changelog for `1.2.0b2`, which is already released, so this shouldn't be there.

We today don't have a process for updating the changelog before a release, but it could definitely be nice to have (https://github.com/twisted/towncrier, or a similar tool, may be a nice fit for instance).

Reply:
oh okay sorry, i just assumed that the one at the top will be something like `UNRELEASED` or whatever

Reply:
should I simply delete it or what?

Reply:
Yes, it will get included whenever a new release occurs.

Reply:
oh, okay cool

Reply:
undone, please check @mkniewallner